README.md

```diff
@@ -31,7 +32,7 @@ My frontend is listening to that websocket, and it updates the progress bar with
           );
 ```
 
-Lucia: "Let's review the code I wrote to make this all happen. It starts with a POST request to my express route. I've attached an event listener to my buttons that fires this request on click. The data, including information about the vote [1], the name of the question, [2] and the timestamp [3] is sent to my route."
+Lucia: "Let's review the code I wrote to make this all happen. It starts with a POST request to my express route. I've attached an event listener to my buttons that fires this request on click. The data, including information about the vote [1], the name of the question, [2] and the timestamp [3] is sent to my route. I'm including the timestamp here in case I want to use stream processing like windowing in a later iteration of the project."
```

Comment:
```suggestion
Lucia: "Let's review the code I wrote to make this all happen. It starts with a POST request to my express route. I've attached an event listener to my buttons that fires this request on click. The data, including information about the vote [1], the name of the question, [2] and the timestamp [3] is sent to my route. I'm including the timestamp here in case I want to use stream processing, say, creating a windowed filter, in a later iteration of the project."
```